### ISSUE TITLE
TypeError: bugsnag_react_1.default is not a function

### ISSUE THREAD

**Issue by patrickomeara** [bugsnag/bugsnag-react#8]
### Expected behavior
react-bugsnag loads

### Observed behavior
Page doesn't render, console has this error
```
[bugsnag] Loaded!
TypeError: bugsnag_react_1.default is not a function
```

### Steps to reproduce
```js
import * as React from 'react';
import bugsnag from 'bugsnag-js';
import { render } from "react-dom";
import createPlugin from 'bugsnag-react';

const bugsnagClient = bugsnag('<redacted>');

const ErrorBoundary = bugsnagClient.use(createPlugin(React));

render(
    <ErrorBoundary>
        test
    </ErrorBoundary>,
    document.getElementById("root"),
);
```

### Version
bugsnag-react v1.0.0
bugsnag-js v4.2.0

**Comment by bengourley** [comment]
Hi @patrickomeara. What bundler are you using (Weback, Rollup, Browserify)? And how are you using it? If you could supply the method you're using to bundle your app it'd help us to reproduce the problem. Thanks!

**Comment by patrickomeara** [comment]
Hi @bengourley I'm using webpack, with `ts-loader`.

```js
// webpack.config.js
const path = require('path');

module.exports = {
  devtool: 'inline-source-map',
  entry: './src/index.tsx',
  output: {
    filename: 'bundle.js',
    path: path.resolve(__dirname, 'dist'),
  },
  resolve: {
    // Add `.ts` and `.tsx` as a resolvable extension.
    extensions: ['.ts', '.tsx', '.js']
  },
  module: {
    rules: [
      // all files with a `.ts` or `.tsx` extension will be handled by `ts-loader`
      { test: /\.tsx?$/, loader: 'ts-loader' }
    ]
  }
}
```

**Comment by bengourley** [comment]
Fixed in `v1.0.1` (via 07a0c4b9f0d27f5e1917e194a46f5e1323174e58). Thanks!
